bundles/org.openhab.binding.bambulab/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="bambulab"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="temperature-channel">
```

Comment:
The `-channel` suffixes are not needed:
```suggestion
	<channel-type id="temperature">
```

Also this channel type could be split into two, one for readings and one for setpoints. This would make it possible to add semantic tags.

Reply:
I do not support setting temperature for channels

Reply:
Right, they are all read-only. I was mislead because there are some target temperature channels, e.g. `nozzleTargetTemperature`. I'm not sure if tags `Temperature`, `Setpoint` is allowed for read-only channels. In this case I guess it's actually a setpoint, it's just that you can't change it?

Reply:
So you want to remove `temperature` category?

Reply:
I guess I'm asking the same question as here: https://github.com/openhab/openhab-addons/issues/12262#issuecomment-1416793650

I'm not suggesting to remove category, only to use two different temperature channel types, one for measurements and one for setpoints (even if they are read-only). Then at least the measurements could be tagged `Measurement`, `Temperature`.

Reply:
Just to be sure:

You propose to have 2 type o channels:

```xml
<channel-type id="temperature-measurment" />
<channel-type id="temperature-setpoint"  />
```
and use `temperature-measurment` for `bed-temperature` and `temperature-setpoint` for `bed-temperature-target` right?

Reply:
Exactly. 👍

Reply:
done, please verify